Bottom: c27d867ad4aa3272562653403c7f0f0398a62b7c
Top:    08cdb9a8cfd429d8b0018d91813c88ace4431342
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-06-10 11:00:44 -0700

Refresh of use_acm

---

diff --git a/src/acm.c b/src/acm.c
index c9ca5b5..c423bb7 100755
--- a/src/acm.c
+++ b/src/acm.c
@@ -45,7 +45,6 @@
 #include <infiniband/ib.h>
 #include <infiniband/sa.h>
 
-#ifdef USE_IB_ACM
 #include <infiniband/acm.h>
 
 #if DEFINE_ACM_MSG
@@ -401,5 +400,3 @@ void ucma_ib_resolve(struct rdma_addrinfo **rai, struct rdma_addrinfo *hints)
 	if (af_ib_support && !(hints->ai_flags & RAI_ROUTEONLY) && (*rai)->ai_route_len)
 		ucma_resolve_af_ib(rai);
 }
-
-#endif /* USE_IB_ACM */
diff --git a/src/cma.h b/src/cma.h
index b575b9b..e6fba8b 100644
--- a/src/cma.h
+++ b/src/cma.h
@@ -162,15 +162,9 @@ extern int af_ib_support;
 
 #define RAI_ROUTEONLY		0x01000000
 
-#ifdef USE_IB_ACM
 void ucma_ib_init();
 void ucma_ib_cleanup();
 void ucma_ib_resolve(struct rdma_addrinfo **rai, struct rdma_addrinfo *hints);
-#else
-#define ucma_ib_init()
-#define ucma_ib_cleanup()
-#define ucma_ib_resolve(x, y)
-#endif
 
 /* Define path record definition if using older version of libibverbs */
 #ifdef DEFINE_PATH_RECORD
